integration-templates/zoho-crm/zoho-crm-accounts.ts

```diff
@@ -0,0 +1,124 @@
+import type { ZohoCRMAccount, NangoSync } from './models';
+
+export default async function fetchData(nango: NangoSync) {
+    try {
+        const responses = await getAllAccounts(nango, '/crm/v2/Accounts');
+        if (responses.length > 0) {
+            const mappedAccounts: ZohoCRMAccount[] = responses.map(mapAccounts);
+            // Save Accounts
+            await nango.batchSave(mappedAccounts, 'ZohoCRMAccount');
```

Comment:
You should batch save the records after every page. Otherwise the sync uses a lot of memory if there are a lot of records in the system (e.g. 100k accounts).